source/sass/pages/ways-to-give-page.scss

```diff
@@ -0,0 +1,42 @@
+.app--ways-to-give {
+  .app__header {
+    background: $color--black;
+    color: $color--white;
+    height: 300px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+
+    .button {
+      display: inline-block;
+      text-transform: uppercase;
```

Comment:
let’s remove this (see https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices#Create_localizable_UI)